src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -186,6 +196,55 @@ def construct_payload(self):
         safe_set(self.session_pool_def, "properties", "dynamicPoolConfiguration", value=dynamic_pool_def)
         safe_set(self.session_pool_def, "properties", "sessionNetworkConfiguration", value=session_network_def)
         safe_set(self.session_pool_def, "properties", "scaleConfiguration", value=session_scale_def)
+        self.set_up_managed_identity_settings()
```

Comment:
If the sessionpool container type is not custom session pool, what will happen?

Reply:
Service will return bad request